### PR TITLE
docs: add a stable diffusion serving example

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingy94@gmail.com>

### CODE REVIEW

**Review by gaocegege — APPROVED** [review]

File: examples/stable_diffusion/server.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+def StableDiffusion(MsgpackMixin, Worker):
+    def __init__(self):
+        self.pipe = StableDiffusionPipeline.from_pretrained(
+            "runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16
+        )
+        self.pipe = self.pipe.to("cuda")
+
+    def forward(self, data):
```
Comment (lkevinzc) [inline examples/stable_diffusion/server.py:40]: what's the type of `data`?

**Comment by lkevinzc** [comment]
@kemingy Can also update README example section?

File: examples/stable_diffusion/server.py (hunk 1 above)
Comment (kemingy) [reply examples/stable_diffusion/server.py:40]: Should be a list of strings. Will update the type hint.

**Comment by kemingy** [comment]
> @kemingy Can also update README example section?

Sure, will also update the doc site.

File: docs/example/stable_diffusion.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+This example provides a demo service for stable diffusion. You can develop this in the container environment by using [envd](https://github.com/tensorchord/envd): `envd up`.
+
+##### Server
+
+    envd build -t sd:serving
```
Comment (lkevinzc) [inline docs/example/stable_diffusion.md:5]: This command should be run under `examples/stable_diffusion`?

Comment (kemingy) [reply docs/example/stable_diffusion.md:5]: Yes, under the same folder with `build.envd` file. Let me add some comments to make it clear.

Comment (lkevinzc) [reply docs/example/stable_diffusion.md:5]: Or if at root, can the command specify the file? Since the server starting command is at root level.

Comment (kemingy) [reply docs/example/stable_diffusion.md:7]: Otherwise you can use `envd build -p examples/stable_diffusion -t sd:serving`

**Review by lkevinzc — APPROVED** [review]

**Pull request merged**